patches/targets-kirkwood.patch

```diff
@@ -0,0 +1,30 @@
+--- a/package/gluon-core/luasrc/lib/gluon/upgrade/010-primary-mac
++++ b/package/gluon-core/luasrc/lib/gluon/upgrade/010-primary-mac
+@@ -119,6 +119,9 @@ local primary_addrs = {
+                        'openmesh,a42',
+                        'openmesh,a62',
+                }},
++               {'kirkwood', 'generic', {
++                       'linksys,e4200-v2',
++               }},
+                {'mpc85xx', 'p1020', {
+                        'aerohive,hiveap-330',
+                        'ocedo,panda',
+--- /dev/null
++++ b/targets/kirkwood-generic
+@@ -0,0 +1,5 @@
++-- Linksys
++
++device('linksys-e4200-v2', 'linksys_e4200-v2', {
++	broken = true,
++})
+--- a/targets/targets.mk
++++ b/targets/targets.mk
+@@ -8,6 +8,7 @@ $(eval $(call GluonTarget,bcm27xx,bcm2709))
+ $(eval $(call GluonTarget,ipq40xx,generic))
+ $(eval $(call GluonTarget,ipq40xx,mikrotik))
+ $(eval $(call GluonTarget,ipq806x,generic))
++$(eval $(call GluonTarget,kirkwood,generic))
```

Comment:
This line has to go further down, into the `BROKEN` section. Similar to https://github.com/freifunk-gluon/gluon/pull/3240/files#diff-84346d32209f957ce32a5d88da1d631fc0e9d165093c75acdd5db04a13c140ffR32

Reply:
Does any human ever see the patched sources? Isn't it only applied in the CI for the build on the fly?

Reply:
As there seems to be more activity in this section, this will mean that you probably have to fix the patch more often.

Reply:
The patch is only applied ad-hoc during the build. Don't worry about maintaining it, we will take care of it

The important part is that it is in the correct section otherwise builds with a disabled BROKEN flag will not work properly.